### ISSUE TITLE
Entropy for multiple arrays

### ISSUE THREAD

**Issue by jakubMitura14** [MattWillFlood/EntropyHub.jl#4]
Hello thanks for sharing your fantastic work! 
I would like to add to my loss function measure of dissimilarity of parameters 
Let's say I have 100  3d arrays that are parameters in neural network and I would like to maximize amount of information and minimize similarity between them.
As I want to add it to loss function entropy measure needs to be differentiable
What can I use ?

**Comment by MattWillFlood** [comment]
Hello @jakubMitura14,
Thanks for your question :)
I'm not sure that I fully understand your question, but in any case, I don't think a solution to your problem can be provided by EntropyHub functions...
This toolkit is primarily for analysing 1D (time series) signals, or 2D (images) arrays. At present, there are no tools in place to estimate the amount of shared info between 3D arrays.
Best of luck!
